module-2/backend/week_four.md

```diff
@@ -10,14 +10,38 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+ A way to maintain state over HTTP request/response cycle.
+
 * What’s the difference between a session and a cookie?
+ A session is similiar to  a cookie except it allows you to have authorization/authentication.
+
 * What’s a flash and when do you want to use flashes?
+ Flash is a piece of functionality you get with rails. It lets you display temporary messages to your users.
+
 * Why do people say “HTTP is stateless”?
+ As a result of being  a RESTful app. It doesn't hold information during communication transactions and after they end.
```

Comment:
Not tied to ReST. ReST is about the pattern that our URLs follow when dealing with resources. Second part of your answer looks good.